salesagility/SuiteCRM
# Fix #9468 - Adding Security Suite subpanels to new custom modules

Closes #9468

## Description
As the Issue describes, newly created custom modules don't have the Security Suite subpanels by default, they need to be added manually in their modules' code.

This PR adds the necessary code to the SugarObjects templates for the different module types that will be later used by the ModuleBuilder when creating the modules. Basically, it adds the definition of the Security Suite subpanels in the same way that the rest of the metadata definition is added.

## Motivation and Context
With this change, newly created modules will have the Security Suite subpanel by default

## How To Test This
1. Create a module in the ModuleBuilder
2. Deploy the module
3. Create a record and check that the SS subpanel appear in their DetailView.

## Types of changes

### Final checklist
